pkg/controller/controller_instance.go

```diff
@@ -1208,41 +1208,31 @@ func (c *controller) pollServiceInstance(instance *v1beta1.ServiceInstance) erro
 			}
 			toUpdate := clone.(*v1beta1.ServiceInstance)
 
+			var (
```

Comment:
I'm not sure what the most go idiomatic way to do this is, but to me this would read cleaner as:
reason := successDeprovisionReason
message := successDeprovisionMessage
if mitigatingOrphan {
  reason = successOrphanMitigationReason
  message = successOrphanMitigationMessage
}

Reply:
Not sure either what is usually preferred. We use switches elsewhere in our code when there's more than 2 possibilities, and here I like it still even though it's a few more lines of code because it is very clear in its intention that there are 2 situations we're handling, with different reason/message each.